native/src/main/java/org/ballerinalang/redis/connection/RedisConnectionManager.java

```diff
@@ -333,7 +333,7 @@ private ServerAddress createServerAddress(String hostStr) throws RedisConnectorE
     }
 
     public void releaseResources(Object redisCommands) {
-        if (!isPoolingEnabled()) {
+        if (!isPoolingEnabled() || redisCommands == null) {
```

Comment:
```suggestion
        if (!isPoolingEnabled() ||  Objects.isNull(redisCommands)) {
```
This is the java standard way of checking nullability of an Object since JDK8.

Reply:
Yeah this is added by mistake. Updated with https://github.com/ballerina-platform/module-ballerinax-redis/pull/200/commits/795bc934617bff1e97c1c37fcd5557385dab64ea